ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionsReader.scala

```diff
@@ -73,25 +74,29 @@ private[dao] final class TransactionsReader(
   private def offsetFor(response: GetTransactionTreesResponse): Offset =
     ApiOffset.assertFromString(response.transactions.head.offset)
 
-  private def deserializeEvent[E](verbose: Boolean)(entry: EventStorageBackend.Entry[Raw[E]])(
-      implicit loggingContext: LoggingContext
+  private def deserializeEvent[E](
+      eventProjectionProperties: EventProjectionProperties
+  )(entry: EventStorageBackend.Entry[Raw[E]])(implicit
+      loggingContext: LoggingContext
   ): Future[E] =
-    entry.event.applyDeserialization(lfValueTranslation, verbose)
+    entry.event.applyDeserialization(lfValueTranslation, eventProjectionProperties)
 
-  private def deserializeEntry[E](verbose: Boolean)(
+  private def deserializeEntry[E](eventProjectionProperties: EventProjectionProperties)(
       entry: EventStorageBackend.Entry[Raw[E]]
   )(implicit loggingContext: LoggingContext): Future[EventStorageBackend.Entry[E]] =
-    deserializeEvent(verbose)(entry).map(event => entry.copy(event = event))
+    deserializeEvent(eventProjectionProperties)(entry).map(event => entry.copy(event = event))
 
   override def getFlatTransactions(
       startExclusive: Offset,
       endInclusive: Offset,
       filter: FilterRelation,
-      verbose: Boolean,
+      eventProjectionProperties: EventProjectionProperties,
   )(implicit loggingContext: LoggingContext): Source[(Offset, GetTransactionsResponse), NotUsed] = {
     val span =
       Telemetry.Transactions.createSpan(startExclusive, endInclusive)(qualifiedNameOfCurrentFunc)
-    logger.debug(s"getFlatTransactions($startExclusive, $endInclusive, $filter, $verbose)")
+    logger.debug(
+      s"getFlatTransactions($startExclusive, $endInclusive, $filter, $eventProjectionProperties)"
+    )
```

Comment:
I am not sure what to do here with the logging... what is important then later to be shown?